app/Http/Controllers/UserPreferencesController.php

```diff
@@ -57,8 +57,11 @@ public function index(Request $request)
             'dashboards' => Dashboard::allAvailable($user)->with('user')->get(),
             'default_dashboard' => UserPref::getPref($user, 'dashboard'),
             'note_to_device' => UserPref::getPref($user, 'add_schedule_note_to_device'),
-            'locale' => UserPref::getPref($user, 'locale') ?: 'en',
+            'locale' => UserPref::getPref($user, 'locale') ?: Config::get('locale'),
```

Comment:
Config::get('local') is not the right setting.  It should be `\config('app.locale', 'en');`

Reply:
changed :-)